sideboard/tests/test_configuration.py

```diff
@@ -4,7 +4,23 @@
 import pytest
 from mock import Mock
 
-from sideboard.config import parse_config, get_config_root
+from sideboard.config import get_config_files, get_config_overrides, \
+    get_config_root, get_module_and_root_dirs, parse_config, uniquify
+
+
+def test_uniquify():
+    assert [] == uniquify([])
+    assert ['a', 'b', 'c'] == uniquify(['a', 'b', 'c'])
+    assert ['a', 'b', 'c', 'd', 'e'] == uniquify(['a', 'b', 'a', 'c', 'a', 'd', 'a', 'e'])
+    assert ['a'] == uniquify(['a', 'a', 'a', 'a', 'a', 'a', 'a', 'a'])
+
+
+@pytest.mark.skipif(
```

Comment:
I don't quite understand this test, since you later set or unset the overrides var during tests. Is the skip here just in case that monkeypatch fails, for some reason?